x/ibc/applications/transfer/handler.go

```diff
@@ -35,6 +38,14 @@ func handleMsgTransfer(ctx sdk.Context, k keeper.Keeper, msg *types.MsgTransfer)
 
 	k.Logger(ctx).Info("IBC fungible token transfer", "token", msg.Token, "sender", msg.Sender, "receiver", msg.Receiver)
 
+	defer func() {
+		telemetry.SetGaugeWithLabels(
+			[]string{"tx", "msg", "ibc", msg.Type()},
+			float32(msg.Token.Amount.Int64()),
```

Comment:
This can panic. All `Int64()` metric calls should be wrapped with a `IsInt64` call. See `x/mint`.